netci.groovy

```diff
@@ -113,11 +114,19 @@ def CreateBuildTasks = { machine, configTag, buildExtra, testExtra, runCodeAnaly
     }
 }
 
-def CreateLinuxBuildTasks = { machine, configTag, linuxBranch, nonDefaultTaskSetup ->
+// Generic task to trigger clang-based cross-plat build tasks
+def CreateXPlatBuildTasks = { machine, platform, configTag, xplatBranch, nonDefaultTaskSetup ->
     [true, false].each { isPR ->
         ['debug', 'test', 'release'].each { buildType ->
-            [true, false].each { staticBuild ->
-                def config = "linux_${buildType}"
+            def staticBuildConfigs = [true, false]
+            if (platform == "osx") {
```

Comment:
No :( osx = platform (that's our internal parameter), OSX = Jenkins parameter. I had to separate them out for consistency with Linux 😞